lib/jekyll-gist/gist_tag.rb

```diff
@@ -4,7 +4,12 @@ class GistTag < Liquid::Tag
 
       def render(context)
         if tag_contents = determine_arguments(@markup.strip)
-          gist_id, filename = tag_contents[0], tag_contents[1]
+          gist_id = context.registers[:page]['gist_id']
```

Comment:
Why not just see if the `gist_id` exists in `context`? Then you could do something like `{% gist page.gist_id %}`